### ISSUE TITLE
Help testing the code

### ISSUE THREAD

**Issue by OlHall** [MaxPonting/Maximum_Engine#1]
Hi Max,
I've compiled your project but Sandbox doesn't appear to do very much?  The main function terminates after creating a couple of objects - it's hard to see what this is demonstrating.
Best regards,
Oliver Hall (Ultamation)

**Comment by MaxPonting** [comment]
Hi Oliver,
Sorry, I was recently doing some unit testing on the sandbox program, the
program terminates after some values are checked. I have pushed a new
version to git that should compile and run properly in Visual Studio. When
you run it, a window will open that renders a square and some performance
metrics. If you want to see more of what the library can do I have a
separate program made using ME that demonstrates its physics engine.
MaxPonting/ME_Physics_Simulation:
A physics simulation made using ME (github.com)
<https://github.com/MaxPonting/ME_Physics_Simulation>. I have also attached
the dissertation I wrote about it.

Thanks,
Max
